tests/shopify_python/test_google_styleguide.py

```diff
@@ -225,3 +225,57 @@ def test_use_cond_exprs(self):
         # root_not_msg should add no messages
         with self.assertNoMessages():
             self.walk(root_not_msg)
+
+    def test_unary_lambda_func(self):
+        unary_root = astroid.builder.parse("""
+        def unaryfnc():
+            unary_pass = map(lambda x: not (x+3), [1, 2, 3, 4])
+            unary_msg = map(lambda x: -x, [1, 2, 3, 4])
+        """)
+
+        # Unary Op Test 1: This will not trigger a message
+        ulam_pass = unary_root.body[0].body[0].value.args[0]
+        with self.assertNoMessages():
+            self.walk(ulam_pass)
+
+        # Unary Op Test 2: This will trigger a message
+        ulam_msg = unary_root.body[0].body[1].value.args[0]
+        unary_message = pylint.testutils.Message('lambda-func', node=ulam_msg, args={
+            'op': 'operator.neg',
+            'lamfun': 'lambda x: - x'
+            })
+        with self.assertAddsMessages(unary_message):
```

Comment:
However you decide to address the test comments for `test_binary_lambda_func`, do the same for `test_unary_lambda_func`